knowledge_graphs/hardware_optimized_config.py

```
@@ -0,0 +1,361 @@
+"""
+Hardware-optimized configuration for i7-3770K, 32GB RAM, RTX 4070 12GB VRAM.
+
+This module provides optimized settings tailored for the specific hardware configuration
+to maximize performance while staying within hardware limits.
+"""
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)